composer.json

```diff
@@ -10,11 +10,11 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0 || ^8.0",
-        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "symfony/yaml": "^3.4 || ^4.3 || ^5.0 || ^6.0"
+        "php": "^7.2 || ^8.0",
```

Comment:
Wasn't sure about 7.2 or 7.1.3 which is the given version in the composer.json in the 4.4 symfony branch.

Reply:
This is great. Thanks.